templates/commands/goldentest/test_funcs.go

```diff
@@ -252,3 +253,24 @@ func renameGitDirsAndFiles(dir string) error {
 	}
 	return nil
 }
+
+func parseTemplateLocations(dir string) ([]string, error) {
```

Comment:
I think we should remove "parse" from this function name. It's not really parsing, it's crawling.